custom-words.txt

```diff
@@ -1072,6 +1072,7 @@ johnsmith
 JSONLD
 Jtoken
 jumpbox
+jwksUri
```

Comment:
this file should be rebased after the PR is merged https://github.com/Azure/azure-rest-api-specs/pull/20890 

`custom-words.txt` was updated by the PR ^^